web-client/src/presenter/actions/setupEditPetitionDetailFormAction.js

```diff
@@ -17,9 +17,11 @@ export const setupEditPetitionDetailFormAction = ({
 
   store.set(state.form, {
     caseType: caseDetail.caseType,
+    hasVerifiedIrsNotice: caseDetail.hasVerifiedIrsNotice,
```

Comment:
Nit: should this be added to the action tests?

Reply:
Yup. Let me update that.

Reply:
Added some tests for these new fields.